angular.json

```diff
@@ -3,7 +3,7 @@
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
-    "chatbot_widget_angular": {
+    "starter-kit_widget_angular": {
```

Comment:
Don't forget to update the `starter-kit_widget_angular` references to `chatbot_widget_angular`. 😉